src/views/tiles/DataSource.svelte

```diff
@@ -1,9 +1,9 @@
 <div class="tile-data">
 	<Tile>
 		<svelte:fragment slot="icon">
-			<div class="tooltip" data-tooltip={dataUserName}>
-				<Avatar size="lg" name={dataUserName} badge={`${datasource.id}`} />
-			</div>
+			{#if datasource.type === 1}
+				<Icon size="2x" color="gray">{@html Cube}</Icon>
```

Comment:
@Valexr hmmm I just realized it will be VERY inefficient on the large pages with 100-500 items... the same SVG will be copied in DOM over and over... May be we could do it as a div with the once pre-defined class?

Reply:
We are use inline strategy for bundle all in one file... if make use svg files as background style or img src, we need copy them to bundle folder & it's not be one file.